metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -303,34 +303,66 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
         ).topics
         extra_topic_details = self.fetch_extra_topic_details(topics.keys())
 
-        for t, t_detail in topics.items():
-            self.report.report_topic_scanned(t)
-            if self.source_config.topic_patterns.allowed(t):
+        for topic, topic_detail in topics.items():
```

Comment:
Thanks for the naming change here, I think it makes the code more readable.

Reply:
sure.